yogstation/code/modules/antagonists/shadowling/special_shadowling_abilities.dm

```diff
@@ -97,6 +97,8 @@
 			H.equip_to_slot_or_del(new /obj/item/clothing/suit/space/shadowling(H), SLOT_WEAR_SUIT)
 			H.equip_to_slot_or_del(new /obj/item/clothing/head/shadowling(H), SLOT_HEAD)
 			H.set_species(/datum/species/shadow/ling) //can't be a shadowling without being a shadowling
+			H.dna.remove_all_mutations(list(MUT_NORMAL, MUT_EXTRA), TRUE)
+			ADD_TRAIT(H, TRAIT_GENELESS, SHADOWLING_TRAIT)
```

Comment:
```suggestion
			ADD_TRAIT(H, TRAIT_GENELESS, SHADOWLING_TRAIT)
```
why not add this to species trait (shadow/ling) instead of defining a trait only for this